css/main.css

```diff
@@ -1,3 +1,861 @@
+        justify-content: flex-start;
+        margin-top: 100px;
+        padding: 0 0 100px 0;
+    }
+
+    .section3-img {
+        grid-area: image;
+        width: 600px;
+        height: 749px;
+        overflow: visible;
+        object-fit: cover;
+
+    }
+
+    .section3-title {
+        text-align: left;
+        padding-right: 10px;
+        margin: 0;
+        font-size: 3.5em;
+        line-height: 1.3em;
+        border: 2px dotted red;
+    }
+
+    .section3-paragraph {
+       font-size: 1.1em;
+       line-height: 1.8em;
+       text-align: left;
+       padding-right: 120px;
+
+    }
+
+    .section3-button {
+        padding: 25px 70px;
+        margin-top: 10px;
+        text-align: center;
+    }
+
+}
+
+.section4-wrapper {
+    background-color: var(--lilac);
+    padding: 20px 25px 30px 25px;
+}
+
+.section4-title {
+    margin-bottom: 12px;
+}
+
+.section4-sub-title {
+    font-size: 1.375rem;
+    padding: 0 30px;
+    margin-top: 15px;
+}
+
+.section4-img1 {
+    width: 280px;
+    height: 274px;
+}
+
+.section4-img2 {
+    width: 280px;
+    height: 274px;
+    margin: 20px 0 5px 0;
+}
+
+.section4-img3 {
+    width: 280px;
+    height: 274px;
+    margin-top: 20px;
+}
+
+.section4-button {
+    font-family: Raleway;
+    font-weight: 300;
+    padding: 12px 50px;
+    margin: 15px 0 10px 0;
+    border: 2px solid var(--black);
+    font-size: 1em;
+    background-color: var(--lylac);   
+}
+
+@media only screen and (min-width: 900px) {
+    .section4-wrapper {
+        margin-top: 100px;
+    }
+
+    .section4-title {
+        font-size: 4em;
+        margin-bottom: 70px;
+    }
+
+    .section4-row-wrapper {
+        /* display: grid;
+        grid-gap: 50px;
+        grid-template-columns: repeat(3, 1fr);
+        grid-template-rows: repeat(1, 1fr); 
+        margin: 10px auto 0 30px;
+        padding: 0 30px;
+        max-width: 100%; */
+        display: flex;
+        flex-direction: row;
+        gap: 20px;
+        border: 2px dotted red;
+        height: 700px;
+        margin-left: 30px;
+    }
+
+    .section4-column-wrapper1 {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: flex-start;
+        text-align: left;
+        padding-right: 40px;
+        font-size: 1.06rem;
+        border: 2px dotted blue;
+        align-self: flex-end;
+    }
+
+    .section4-column-wrapper2 {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: flex-start;
+        text-align: left;
+        padding-right: 40px;
+        font-size: 1.06rem;
+        border: 2px dotted blue;
+    }
+
+    .section4-img1 {
+        width: 310px;
+        height: 390px
+    }
+
+    .section4-img2 {
+        width: 310px;
+        height: 390px;
+        margin: 0;
+        padding: 0;
+    }
+
+    .section4-img3 {
+        width: 310px;
+        height: 390px;
+        margin: 0;
+        padding: 0;
+    }
```

Comment:
This could be written as
`.section4-img1,
.section4-img2,
.section4-img3 {
        width: 310px;
        height: 390px;
        margin: 0;
        padding: 0;
    }`